src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Security.TransportSecurity.Tests.csproj

```diff
@@ -1,44 +1,48 @@
-            <Name>System.Private.ServiceModel</Name>
-            <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-            <OutputItemType>Content</OutputItemType>
-            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-            <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
-        </ProjectReference>
-        <ProjectReference Include='$(WcfScenarioTestCommonProj)'>
-            <Project>{9098B41C-9C2E-4FC6-B7D8-FC3411736A22}</Project>
-            <Name>ScenarioTests.Common</Name>
-        </ProjectReference>
-    </ItemGroup>
-    <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
\ No newline at end of file
+  <PropertyGroup>
+    <TestCategories>OuterLoop</TestCategories>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <MinimumVisualStudioVersion>11.0</MinimumVisualStudioVersion>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Security.TransportSecurity.Tests</RootNamespace>
+    <AssemblyName>Security.TransportSecurity.Tests</AssemblyName>
+    <SignAssembly>false</SignAssembly>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ProjectGuid>{452BA2A7-81AD-4146-AD02-35EC242FF0EB}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Https\ClientCredentialTypeTests.cs" />
+    <Compile Include="Https\HttpsTests.cs" />
+    <Compile Include="Tcp\ClientCredentialTypeTests.cs" />
+    <Compile Include="Tcp\IdentityTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="project.json" />
+  </ItemGroup>
+  <ItemGroup>
+    <!-- Compile tests against the contract, but copy our local-built implementation for testing -->
+    <ProjectReference Include="$(WcfSourceProj)">
+      <Project>{9e50e7bf-cd6e-4269-a6dd-59fd0bd6c0fd}</Project>
+      <Name>System.Private.ServiceModel</Name>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <OutputItemType>Content</OutputItemType>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
+    </ProjectReference>
+    <ProjectReference Include='$(WcfScenarioTestCommonProj)'>
+      <Project>{9098B41C-9C2E-4FC6-B7D8-FC3411736A22}</Project>
+      <Name>ScenarioTests.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="$(WcfInfrastructureCommonProj)">
+      <Project>{AFD69665-89A3-42AE-A32E-AB2CBBE6EE7E}</Project>
+      <Name>Infrastructure.Common</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+</Project>
```

Comment:
If there are new project references here, they will also need to be added in TFS to the ToF wrapper projects.  Check with @hongdai who might be doing the same update.  I believe the change to TFS can be made first safely so that when this PR is merged, TFS is already correct.